component/input/input.html

```diff
@@ -0,0 +1,147 @@
+                                fill="currentColor"></path>
+                        </svg></a>
+                </li>
+                <li class="social-links">
+                    <a href="https://twitter.com/web_AshishSethi" target="_blank"><svg
+                            xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+                            aria-hidden="true" role="img" class="iconify iconify--mdi" width="32" height="32"
+                            preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24">
+                            <path
+                                d="M22.46 6c-.77.35-1.6.58-2.46.69c.88-.53 1.56-1.37 1.88-2.38c-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29c0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15c0 1.49.75 2.81 1.91 3.56c-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07a4.28 4.28 0 0 0 4 2.98a8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21C16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56c.84-.6 1.56-1.36 2.14-2.23z"
+                                fill="currentColor"></path>
+                        </svg></a>
+                </li>
+            </ul>
+        </div>
+
+        <div class="main-content">
+
+            <h1>Input</h1>
+            <p>Inputs boxes are used to take any kind of input from the user like name, e-mail, phone number, date of birth etc.</p>
+            
+            <h2>Textbox Input</h2>
+            <p> You can use Input box for sign in/ sign up or for taking data from the user.</p>
+            
+            <div class="img-container">
+                
+                <input type="text" class="input-box" placeholder="Enter email address" id="email">
+                <input type="password" class="input-box" placeholder="Enter password">
+                <input type="date" class="input-box" placeholder="Enter date of birth">
+                <input type="file" class="input-box" placeholder="upload file">
+            </div>
+            
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=base16-dark&wt=none&l=htmlmixed&width=955&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cinput%2520type%253D%2522text%2522%2520class%253D%2522input-box%2522%2520placeholder%253D%2522Enter%2520email%2520address%2522%253E%250A%253Cinput%2520type%253D%2522password%2522%2520class%253D%2522input-box%2522%2520placeholder%253D%2522Enter%2520password%2522%253E%250A%253Cinput%2520type%253D%2522date%2522%2520class%253D%2522input-box%2522%2520placeholder%253D%2522Enter%2520date%2520of%2520birth%2522%253E%250A%253Cinput%2520type%253D%2522file%2522%2520class%253D%2522input-box%2522%2520placeholder%253D%2522upload%2520file%2522%253E"
+  style="width: 955px; height: 148px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
would suggest to add a class for styling iframe to maintain uniformity .
Rest LGTM

Reply:
done! Thanks